### PR TITLE
PR for Home page

### PR DESCRIPTION
**Pull request by kiranpratyush**
Added Home page to my eCommerce site. Please review and give feedback(Note: This site is not responsive yet)
[Testing link](https://testing2322.netlify.app/index.html)

### CODE REVIEW

File: general.css [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+}
+.header-bottom {
+  display: flex;
+  justify-content: space-between;
+  padding: 15px;
+  padding-left: 100px;
+  padding-right: 100px;
+  background-color: var(--color-white);
+  align-items: center;
+}
```
Comment (BelideNikhil) [inline general.css]: header-top and bottom are same, so try re-using the lines.

Comment (kiranpratyush) [reply general.css]: I have updated it Thank you for your suggestions. Can you please approve it

**Review by BelideNikhil — APPROVED** [review]
Great work!
Loved the homepage!

**Comment by kiranpratyush** [comment]
> Great work! Loved the homepage!

Thank you

**Review by techieshubham30 — APPROVED** [review]

**Pull request merged**